libs/build_tiff.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
What is tiff used for?

Reply:
The tiff library is required for building  "proj" library. Modified the order of libraries requested to be built in build_stack.sh to reflect this dependency, such that:
...
build_lib sqlite
build_lib tiff
build_lib proj
...